examples/discovery-search-app/cypress/integration/collection_filter/spec.ts

```diff
@@ -0,0 +1,142 @@
+describe('Collection Filter', () => {
+  beforeEach(() => {
+    //TODO: swap this section with the helper function, once that code is merged in
+    // Sets up and handles the collections, component settings, and initial query requests that run on page-load
+    cy.server();
+    cy.fixture('collections/collections.json').as('collectionsJSON');
+    cy.route('GET', '**/collections?version=2019-01-01', '@collectionsJSON').as('getCollections');
+    cy.fixture('component_settings/componentSettings.json').as('componentSettingsJSON');
+    cy.route('GET', '**/component_settings?version=2019-01-01', '@componentSettingsJSON').as(
+      'getComponentSettings'
+    );
+    cy.fixture('query/query.json').as('queryJSON');
+    cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('postQuery');
+    cy.visit('/');
+    cy.wait(['@getCollections', '@getComponentSettings', '@postQuery']);
+
+    // Set up/override routes & fixtures that are specific to this file
+    cy.fixture('query/singleCollectionQuery.json').as('singleCollectionQueryJSON');
+    cy.fixture('query/doubleCollectionQuery.json').as('doubleCollectionQueryJSON');
+  });
+
+  describe('when the example app loads', () => {
+    it('the collection facet select does not appear', () => {
+      cy.get('.collection-facet-select').should('not.exist');
+    });
+  });
+
+  describe('when a query is made', () => {
+    beforeEach(() => {
+      cy.get('.bx--search-input').type('abil{enter}');
+      cy.wait('@postQuery');
+    });
+
+    it('the collection selector should appear', () => {
+      cy.get('#collection-facet-select').should('exist');
```

Comment:
Would cypress testing library effect the way you could select these collection facets?

Reply:
It definitely could. I think we could swap this out for
```
cy.getById('collection-facet-select').should('exist');
```

Or, we could use some other parameter to find this element by.